google-cloud-storage/src/test/java/com/google/cloud/storage/BlobTest.java

```diff
@@ -662,4 +661,25 @@ public Long answer() throws Throwable {
     byte actual[] = Files.readAllBytes(file.toPath());
     assertArrayEquals(expected, actual);
   }
+
+  @Test
```

Comment:
Per Google practices, new tests should use Mockito, not EasyMock. Yes, even if this means some tests use Mockito and others use EasyMock.

Reply:
Is this rule applied to the tests within a single file or withing a package? 
Is it possible to make an exception?